app/controllers/orders/client_summary.js

```diff
@@ -17,29 +21,78 @@ export default detail.extend({
 
   identityTypes: Ember.computed(function() {
     return [
-      { name: 'Hong Kong Identity Card', id: 1 },
-      { name: 'Asylum Seeker Recognizance Form', id: 2}
+      { name: "Hong Kong Identity Card", id: 1 },
+      { name: "Asylum Seeker Recognizance Form", id: 2 }
     ];
   }),
 
+  isLongDescription: Ember.computed("model.purposeDescription", function() {
+    return this.get("model.purposeDescription.length") > LONG_DESCRIPTION_SIZE;
+  }),
+
+  toggleShowMore: Ember.observer("isLongDescription", function() {
+    const isLongDescription = this.get("isLongDescription");
+    if (isLongDescription) {
+      this.set("showMore", true);
+    } else {
+      this.set("showMore", false);
+    }
+  }),
+
+  shortPurposeDescription: Ember.computed(
+    "showInlineEdit",
+    "showMore",
+    function() {
+      let purposeDescription = this.get("model.purposeDescription");
+      if (this.get("showMore")) {
```

Comment:
shouldn't it be the opposite ? "show**More**"  should be be showing more, but here's it's cutting the text 🤔